ext/background.js

```diff
@@ -607,4 +607,20 @@
       message, sender, sendResponse
     ).includes(true);
   });
+
+  /* External message passing */
+  browser.runtime.onMessageExternal.addListener(
+    // we could theoretically whitelist extension ids we agree
+    // to communicate with (if it's not additional overhead)
+    // e.g if (whitelistedExtensions.includes(sender.id)){ handle and respond }
+    (message, sender, sendResponse) =>
+    {
+      // for now handle only prefs and stats requests
+      if (message.type === "prefs.get" || message.type.test(/stats./))
```

Comment:
Inside a regular expression `.` is a wild card, so we probably want `\.` instead.

Reply:
You seem to be attempting to access [RegExp.prototype.test](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test) from the string, which I don't think will work. In other words, I think you meant `/stats./.test(message.type)`

Also `/stats./.test(...)` would match anything which contains `"stats"` with at least one character afterwards. It wouldn't even need to start with `"stats"`. For example, I think `"examplestats!"` would match!